docs/docs/command-line-interface.mdx

```diff
@@ -6,6 +6,31 @@ description: Command line interface for open source chatbot framework Rasa. Lear
 abstract: The command line interface (CLI) gives you easy-to-remember commands for common tasks. This page describes the behavior of the commands and the parameters you can pass to them.
 ---
 
+## Log Level
+
+Rasa produces log messages at several different levels (eg. warning, info, error and so on). You can control which level of logs you would like to see with `--verbose` (same as `-v`) or `--debug` (same as `-vv`) as optional command line arguments. See each command below for more explanation on what these arguments mean.
+
+In addition to CLI arguments, several environment variables allow you to control log output in a more granular way. With these environment variables, you can configure log levels for messages created by external libraries such as matplotlib, pika, and kafka. Currently following environment variables are supported:
+
+1. LOG_LEVEL_LIBRARIES: This is the general environment variable to configure log level for all libraries.
+2. LOG_LEVEL_MATPLOTLIB: This is the specialized environment variable to configure log level only for Matplotlib.
+3. LOG_LEVEL_RABBITMQ: This is the specialized environment variable to configure log level only for AMQP libraries, at the moment it handles log levels from `aio_pika` and `aiormq`.
+4. LOG_LEVEL_KAFKA: This is the specialized environment variable to configure log level only for kafka.
+
+General configuration (`LOG_LEVEL_LIBRARIES`) has less priority than library level specific configuration (`LOG_LEVEL_MATPLOTLIB`, `LOG_LEVEL_RABBITMQ` etc); and CLI parameter has the highest precedence. This means variables can be used together with a predictable result. As an example:
```

Comment:
Does CLI parameter have the highest precedence really? In the example below the cli is set to `DEBUG` but does't override the `WARNING` level of matplotlib... or am I understanding the example incorrectly?
Or maybe it only overrides to a _less_ verbose level?

Reply:
Good catch, CLI parameter sets the level at the root level (which also has the important handler - coloredlogs handler). This means CLI level will be used for every log which is not explicitly set (matplotlib, rabbitmq, or rasa and any other library) but it also means CLI level overrides all other log levels if it is a less verbose level, like you said.

I'll change the wording in the docs.